curriculum/challenges/english/25-front-end-development/review-html-accessibility/671a87a39b88245a579c2271.md

```diff
@@ -9,7 +9,44 @@ dashedName: review-html-accessibility
 
 Review the concepts below to prepare for the upcoming quiz.
 
+## Introduction to Accessibility
 
+- **Web Content Accessibility Guidelines**: The Web Content Accessibility Guidelines (WCAG) are a set of guidelines for making web content more accessible to people with disabilities. The four principles of WCAG are POUR which stands for Perceivable, Operable, Understandable, and Robust.
+
+## Assistive Technology for Accessibility
+
+- **Screen readers**: software programs that read the content of a computer screen out loud. They are used by people who are blind or visually impaired to access the web.
+- **Large text or braille keyboards**: used by people with visual impairments to access the web.
+- **Screen magnifiers**: software programs that enlarge the content of a computer screen. They are used by people with low vision to access the web.
+- **Alternative pointing devices**: used by people with mobility impairments to access the web. This includes devices such as joysticks, trackballs, and touchpads.
+- **Voice recognition**: used by people with mobility impairments to access the web. It allows users to control a computer using their voice.
+
+## Accessibility Auditing Tools
+
+- **Common Accessibility Tools**: Google Lighthouse, Wave, IBM Equal Accessibility Checker, and axe DevTools are some of the common accessibility tools used to audit the accessibility of a website.
+
+## Accessibility Best Practices
+
+- **Proper heading level structure**: You should use proper heading levels to create a logical structure for your content. This helps screen readers and other assistive technologies understand the content of your website.
```

Comment:
```suggestion
- **Proper heading level structure**: You should use proper heading levels to create a logical structure for your content. This helps people using screen readers and other assistive technologies understand the content of your website.
```

I apologize for being so picky here. I just want to add the emphasis that this helps people, not technology.